x/evm/keeper/grpc_query_test.go

```diff
@@ -236,10 +236,8 @@ func (suite *KeeperTestSuite) TestQueryStorage() {
 					Address: suite.address.String(),
 					Key:     ethcmn.Hash{}.String(),
 				}
-				exp := &types.QueryStorageResponse{Value: "0x0000000000000000000000000000000000000000000000000000000000000000"}
-				expValue = exp.Value
 			},
-			true,
+			false,
```

Comment:
it took me a few minutes to realize this mean `expectedToPass` - seems like a cosmos-sdk/tendermint thing.